### PR TITLE
DM-43412: Add USDF rapid-analysis deployment

### PR DESCRIPTION
**Pull request by ctslater**
This is the branch we used for operations rehearsal 3, no significant modifications have been made since then.

### CODE REVIEW

File: services/rapid-analysis/values-usdf-comcamsim.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+namespace: rubintv-comcamsim
+image:
+  repository: lsstts/rapid-analysis
+  tag: c0035_usdf
```
Comment (sebastian-aranda) [inline services/rapid-analysis/values-usdf-comcamsim.yaml:4]: There is a more recent tag for the `lsstts/rapid-analysis` image you could use: [c0037_usdf](https://hub.docker.com/layers/lsstts/rapid-analysis/c0037_usdf/images/sha256-27522a49624ca39da6fedec0e529c4d4a7dcaf84c8140da3cad9ae8487da7b83?context=explore).

File: services/rapid-analysis/values-usdf-comcamsim.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  pullPolicy: Always
+location: USDF
+env:
+  http_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  https_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  squid_proxy: http://sdfproxy.sdf.slac.stanford.edu:3128
+  no_proxy: .slac.stanford.edu
+  HTTP_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
+  HTTPS_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
+  SQUID_PROXY: http://sdfproxy.sdf.slac.stanford.edu:3128
```
Comment (sebastian-aranda) [inline services/rapid-analysis/values-usdf-comcamsim.yaml:14]: I wonder why all these urls are the same. Are they changing in the future? if that's so maybe you could add a comment with a reference to a ticket which will handle that work, like a "to do".

Comment (sebastian-aranda) [reply services/rapid-analysis/values-usdf-comcamsim.yaml:14]: also, I assume it is ok that https variables don't use `https://` yet.

**Review by sebastian-aranda — APPROVED** [review]
Nice work @ctslater , this looks pretty good. I left a couple comments, but I'll just approve the PR :+1:.

File: services/rapid-analysis/values-usdf-comcamsim.yaml (hunk 2 above)
Comment (ctslater) [reply services/rapid-analysis/values-usdf-comcamsim.yaml:14]: Yeah it's kind of annoying to have to include all of these, but different programs look for slightly different environment variables for the proxy setting, so USDF usually sets all of them just to be sure.

File: services/rapid-analysis/values-usdf-comcamsim.yaml (hunk 1 above)
Comment (ctslater) [reply services/rapid-analysis/values-usdf-comcamsim.yaml]: Updated to c0037_usdf.

**Pull request merged**